doc/website/advanced/iceoray-on-32-bit.md

```diff
@@ -56,14 +56,15 @@ struct Foo {
 };
 ```
 
-As long as the bitness it the same, one does not have to care and but with shared memory, both applications have to agree to a common layout, else the applications will misbehave.
+As long as the bitness it the same, one does not have to care.
```

Comment:
What do you think to replace the following two sentences with:

As long as the applications share the same bitness, there is no need for special consideration. However, when connecting 32-bit and 64-bit applications via shared memory, both must adhere to a common memory layout. If the layout differs, it can lead to unpredictable behavior and errors in the applications.

Reply:
Sounds great